pkg/config/schema.go

```diff
@@ -77,16 +93,64 @@ var (
 			[]field.SchemaField{WorkspacesField},
 		),
 	}
-	ConfigurationSchema = field.NewConfiguration(
-		configurationFields,
-		field.WithConstraints(fieldRelationships...),
-	)
+	fieldGroups = []field.SchemaFieldGroup{
+		{
+			Name:        "OAuth",
```

Comment:
name should be in this format, similar to field names 

example:
https://github.com/ConductorOne/baton-github-test/blob/main/pkg/config/config.go#L89

Reply:
looks good, thank you

in c1, we will have to map the field group names, similar to fields 
https://github.com/ductone/c1/blob/badc8d4244af453766a7d0b90486fc577a7e34a9/pkg/builtin_connectors/builtin_deployment.go#L363